package.json

```diff
@@ -9,15 +9,21 @@
   "scripts": {
     "init-dir": "sequelize init",
     "migrate": "yarn sequelize-cli db:migrate",
-    "down": "yarn sequelize-cli db:migrate:undo",
+    "down": "yarn sequelize-cli db:migrate:undo:all",
     "seed": "yarn sequelize-cli db:seed:all",
     "unseed": "yarn sequelize-cli db:seed:undo:all",
+    "create:roles": "yarn sequelize db:seed --seed 20220411084040-Role.js",
+    "create:admin": "yarn run build && yarn create:roles && node lib/config/init.js",
     "build": "babel src -d lib",
-    "start": "yarn run build && cross-env NODE_ENV=production node lib/index.js",
-    "dev": "cross-env NODE_ENV=development nodemon --exec babel-node src/index.js",
+    "start": "yarn run build && cross-env NODE_ENV=production yarn migrate && NODE_ENV=production yarn create:admin && cross-env NODE_ENV=production node lib/index.js",
+    "dev": " yarn migrate && yarn create:admin && nodemon --exec babel-node -r dotenv/config src/index.js",
     "lint": "eslint src/** test/** --fix",
     "prepare": "husky install",
-    "test": "yarn migrate --env test && cross-env NODE_ENV=test c8 mocha --timeout 100000 --exit --require @babel/register"
+    "test": "yarn migrate --env test && cross-env NODE_ENV=test yarn create:admin && cross-env NODE_ENV=test c8 --reporter=text --reporter=lcov ./node_modules/.bin/mocha --timeout 200000 --exit --require @babel/register"
```

Comment:
Calling the `create:admin` script is unnecessary since you're going to want to clear the database after each test suite, as you're already calling the function to create an admin when needed in the **test/roles.test.js** file

Reply:
@niyojuste the admin must be created before the db is run regardless of my test other tests are using that.